packages/gafl-webapp-service/src/routes/__tests__/back-links.spec.js

```diff
@@ -204,84 +199,142 @@ describe('The licence-start-time page', () => {
 
 describe('The address-lookup page', () => {
   const currentPage = journeyDefinition.find(currentPage => currentPage.current.page === ADDRESS_LOOKUP.page)
-  it('has a back-link to the licence-summary page if the contact summary has not been seen', () => {
-    expect(currentPage.backLink({})).toBe(LICENCE_SUMMARY.uri)
+  it('has a back-link to the licence-summary page if the contact summary has not been seen', async () => {
+    const status = {}
+    const result = await currentPage.backLink(status)
+    expect(result).toBe(LICENCE_SUMMARY.uri)
+  })
+  it('has a back-link to the contact-summary page if the contact-summary is seen', async () => {
+    const status = {
+      fromSummary: CONTACT_SUMMARY_SEEN
+    }
+    const result = await currentPage.backLink(status)
+    expect(result).toBe(CONTACT_SUMMARY.uri)
   })
-  it('has a back-link to the contact-summary page if the contact-summary is seen', () => {
-    expect(currentPage.backLink({ fromSummary: CONTACT_SUMMARY_SEEN })).toBe(CONTACT_SUMMARY.uri)
+  it('has a back-link to contact details if the contact-details page is seen', async () => {
+    const status = {
+      fromContactDetailsSeen: CHANGE_CONTACT_DETAILS_SEEN.SEEN
+    }
```

Comment:
Small stylistic point, but there's no need to separate status onto a separate line where it's not asserted on, it just gets passed to backLink (same goes for the previous two too)